spec/models/manageiq/providers/automation_manager_spec.rb

```diff
@@ -0,0 +1,4 @@
+describe ManageIQ::Providers::AutomationManager do
+  let(:provider)           { FactoryGirl.build(:provider) }
+  let(:automation_manager) { FactoryGirl.build(:automation_manager, :provider => provider) }
+end
```

Comment:
@chrisarcand Thanks, but this 🍖 is still very raw, not ready for consumption yet 😄. Way too many broken cases. Actively updated...